packages/backend/package.json

```diff
@@ -99,6 +99,12 @@
           "description": "The verbosity of logging according to the following order: trace > debug > info > warn > error > fatal > off.",
           "scope": "resource"
         },
+        "ApplicationWizard.HideGenerator": {
+          "type": "string",
+          "default": "",
+          "description": "Hide the generator from the list of generators. Generator name should be in the format of namespace:generator-name. Generators should be seperated by comma.",
```

Comment:
seperated --> separated
Do you think Paola should review this string?

Reply:
does she review all strings of settings?